hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieStreamSourceV1.scala

```diff
@@ -54,6 +55,9 @@ class HoodieStreamSourceV1(sqlContext: SQLContext,
                            writeTableVersion: HoodieTableVersion)
   extends Source with Logging with Serializable with SparkAdapterSupport {
 
+  private lazy val useNewParquetFileFormat = parameters.getOrElse(HoodieReaderConfig.FILE_GROUP_READER_ENABLED.key(),
```

Comment:
nit: rename the variable for clarity?
```suggestion
  private lazy val enableFileGroupReader = parameters.getOrElse(HoodieReaderConfig.FILE_GROUP_READER_ENABLED.key(),
```